### ISSUE TITLE
Multi-Vehicle Simulation with Gazebo and ROS

### ISSUE THREAD

**Issue by pitergit95** [PX4/PX4-Autopilot#19044]
Hi,

I'm trying to add new iris drone to my multi-vehicle simulation in Gazebo with ROS but i cannot find in latest Firmware the file which is pointed in documentation. It should be here(path): PX4-Autopilot/posix-configs/SITL/init/test/. I can find this file in oldest branch from 2018 but in the latest it is missing. There could be a mismatch between documentation and latest branch. Could you please help me and point where i can add new drone to simulation?

Thank you in advance

**Comment by Jaeyoung-Lim** [comment]
@pitergit95 Which documentation are you referring to?

**Comment by pitergit95** [comment]
Yes, that was the path which i was searching for file to add new drone. Still i cannot find file mentioned here in newest firmware release: 
_create a startup file, and change the file as follows:
make a copy of an existing iris rcS startup file (iris_1 or iris_2) and rename it iris_3_
Where i can find this rcS file?

**Comment by Siri2K** [comment]
Hey @pitergit95, I think this might help you: [multi_vehicle_simulation](https://docs.px4.io/main/en/sim_gazebo_classic/multi_vehicle_simulation.html)
